src/scss/themes/_neucrystalgrape.scss

```diff
@@ -0,0 +1,29 @@
+@use "../utils/neutheme";
+
+// $name
```

Comment:
Can you please remove all these comments and use variables like the other themes?

There are a lot of repeated colors in the theme below.